airflow/config_templates/config.yml

```diff
@@ -1821,6 +1821,17 @@ webserver:
       type: boolean
       example: ~
       default: "False"
+    allow_html_in_dag_docs:
```

Comment:
Not sure about the name of this. It implies its only dag docs, but it covers more than that. Maybe `allow_unsafe_html`?

Reply:
Renamed it to `allow_raw_html_descriptions` - hope this is also OK?

Reply:
I don't like the "descriptions" part since it also controls the custom form. `allow_raw_html`?

Reply:
On the other hand, the only non-deprecated behavior it controls is the descriptions. I feel like we need to at least mention in the description of this config that it also allows the old _html variants to work?